.github/workflows/build-and-push-components.yml

```
@@ -22,6 +22,35 @@ on:
         type: string
         description: 'Name of the Docker Bake group of targets'
         default: default
+  workflow_dispatch:
+    inputs:
+      git_ref:
+        required: false
+        type: string
+        description: 'The branch, tag or SHA to build.'
+        default: ''
+      image_tag:
+        required: true
+        type: string
+        description: 'Image tag to use.'
```

```suggestion
        description: 'Image tag to use.'
        default: latest
```

I think the default could still be `latest`

---

Done, thank you.